src/x86/uarch.c

```diff
@@ -168,6 +170,9 @@ enum cpuinfo_uarch cpuinfo_x86_decode_uarch(
 						case 0x7E: // Ice Lake-U
 							return cpuinfo_uarch_sunny_cove;
 
+						case 0x8C: // Tiger U
```

Comment:
note 8c = 140 is the one that sde -tgl exposes but https://en.wikichip.org/wiki/intel/cpuid confirms 141